public/source/index.php

```diff
@@ -540,7 +550,7 @@
   "access_token": "XXXXXX",
   "token_type": "Bearer",
   "scope": "create update delete",
-  "me": "https://user.example.net/"
+  "me": "https://user.example.net/",
```

Comment:
Don't think this was needed (and it's now invalid JSON), unless we're going to add a `refresh_token`
```suggestion
  "me": "https://user.example.net/"
```

Reply:
I think that's what I meant to do....then realized it is optional so maybe I shouldn't.